### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by godunko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,9 +42,15 @@ It will build `.obj/server/ada_language_server` file.
 
 ### Dependencies
 
-To build the language server you need at least a version of the GNAT compiler,
-and the [Libadalang](https://github.com/AdaCore/libadalang) library to be built
-and available via the `GPR_PROJECT_PATH`.
+To build the language server you need:
+
+ * GNAT compiler
```
Comment (AnthonyLeonardoGracio) [inline README.md:47]: A GNAT compiler

Comment (AnthonyLeonardoGracio) [reply README.md:47]: The VSS library

File: README.md [hunk 2]
```diff
@@ -42,9 +42,15 @@ It will build `.obj/server/ada_language_server` file.
...
+ * [Libadalang](https://github.com/AdaCore/libadalang) library, it should be
```
Comment (AnthonyLeonardoGracio) [inline README.md:48]: "(it should be built)" instead of ", it should be build"

File: README.md [hunk 3]
```diff
@@ -115,7 +115,7 @@ The `ada_language_server` doesn't require/understand any command line options.
 | `documentLink/resolve`                |                    |
 | `textDocument/documentColor`          |                    |
 | `textDocument/colorPresentation`      |                    |
-| `textDocument/formatting`             |                    |
+| `textDocument/formatting`             | :white_check_mark: |
 | `textDocument/rangeFormatting`        |                    |
```
Comment (AnthonyLeonardoGracio) [inline README.md:119]: rangeFormatting has also ben implemented actually

Comment (reznikmm) [reply README.md:119]: No, it's disabled now. We reverted `rangeFormatting` because it caused a few regressions in GS testsuite.

**Review by AnthonyLeonardoGracio — COMMENTED** [review]
these commits should not be under this PR right?

**Comment by godunko** [comment]
It is rebase of master branch, don't know why they are here :(

File: README.md [hunk 4]
```diff
@@ -42,9 +42,15 @@ It will build `.obj/server/ada_language_server` file.
...
 ### Dependencies
 
-To build the language server you need at least a version of the GNAT compiler,
-and the [Libadalang](https://github.com/AdaCore/libadalang) library to be built
-and available via the `GPR_PROJECT_PATH`.
+To build the language server you need:
+
+ * A GNAT compiler
+ * [Libadalang](https://github.com/AdaCore/libadalang) library (it should be
+   build)
```
Comment (AnthonyLeonardoGracio) [inline README.md:49]: 'built', not 'build'

**Review by AnthonyLeonardoGracio — CHANGES_REQUESTED** [review]

**Review by AnthonyLeonardoGracio — APPROVED** [review]

**Pull request merged**